simphony/cuds/lattice.py

```diff
@@ -67,6 +69,43 @@ def origin(self):
         return self._origin
 
     def get_node(self, index):
+        pass
```

Comment:
**Querstion**: These methods are already defined in ABCLattice, redefining them here into empty overrides the requirements for them to be defined and does not provide any additional functionality. It is not clear why this is necessary.